build/conda/installer/assets/installer_welcome.txt

```diff
@@ -1,6 +1,13 @@
-Welcome to the OpenBB Installer.
+Welcome to the OpenBB Platform Installer.
 
 
-This application will install Python and the latest version of OpenBB Platform on your computer.
+This application will install the latest version of the OpenBB Platform on your computer in a self-contained, Python 3.12, environment.
+
+Several launch scripts are included; CLI, Fast API, Jupyter, iPython, updater, and other shortcuts.
```

Comment:
Let's fix up the names
- OpenBB CLI
- OpenBB API
- Jupyter Notebook (specify Notebook)
- IPython (capital i)